CHANGELOG.md

```diff
@@ -30,6 +21,26 @@ New Features
   localParticipant.removeTracks(tracks, false);
   ```
 
+Bug Fixes
+---------
+
+- twilio-video.js can now be used alongside adapter.js. twilio-video.js checks
+  whether or not an RTCPeerConnection implementation supports the "track" event
+  by checking for an `ontrack` property on the RTCPeerConnection. adapter.js
+  sets this property; however it only dispatches "track" events _if_ a function
+  is assigned to the `ontrack` property, meaning that event handlers attached
+  with `addEventListener` will never fire. We now work around this issue by
+  assigning a function to `ontrack`.
+
+Known Issues
+------------
+
+- In Firefox 55 (beta), Participants who connect to a Group Room will not see each
```

Comment:
@markandrus I think this can be removed as the VMS team deployed a fix (after confirming with simpler-signaling, of course).